packages/terra-clinical-label-value-view/src/terra-dev-site/doc/example/LabelValueViewTextTermAndDef.jsx

```diff
@@ -8,6 +8,11 @@ const LabelValueViewText = () => (
         #dl-wrapper > dd {
           margin-bottom: 14px;
         }
+
+        #dl-wrapper {
+          margin-bottom: 0;
+          margin-top: 0;
+        }
```

Comment:
Why do we need to add a style on the wrapper element if we are updating our sass file?

Reply:
This is being applied to the `<dl>` element on line 19 for this specific example and not the one in LableValueView that had its sass file updated. This change is maybe not needed, as it is just an example, but I added it to be consistent in removing these default margins whenever we use a definition list.